docs/content/docs/tutorials/ec2-example.md

```diff
@@ -0,0 +1,450 @@
+kind: VPC
+metadata:
+  name: tutorial-vpc
+spec:
+  cidrBlocks: 
+  - 10.0.0.0/16
+  enableDNSSupport: true
+  enableDNSHostnames: true
+  tags:
+    - key: name
+      value: vpc-tutorial
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: InternetGateway
+metadata:
+  name: tutorial-igw
+spec:
+  vpcRef:
+    from:
+      name: tutorial-vpc
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: NATGateway
+metadata:
+  name: tutorial-natgateway1
+spec:
+  subnetRef:
+    from:
+      name: tutorial-public-subnet1
+  allocationRef:
+    from:
+      name: tutorial-eip1
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: ElasticIPAddress
+metadata:
+  name: tutorial-eip1
+spec:
+  tags:
+    - key: name
+      value: eip-tutorial
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: RouteTable
+metadata:
+  name: tutorial-public-route-table
+spec:
+  vpcRef:
+    from:
+      name: tutorial-vpc
+  routes:
+  - destinationCIDRBlock: 0.0.0.0/0
+    gatewayRef:
+      from:
+        name: tutorial-igw
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: RouteTable
+metadata:
+  name: tutorial-private-route-table-az1
+spec:
+  vpcRef:
+    from:
+      name: tutorial-vpc
+  routes:
+  - destinationCIDRBlock: 0.0.0.0/0
+    natGatewayRef:
+      from:
+        name: tutorial-natgateway1
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: Subnet
+metadata:
+  name: tutorial-public-subnet1
+spec:
+  availabilityZone: us-west-2a
+  cidrBlock: 10.0.0.0/20
+  mapPublicIPOnLaunch: true
+  vpcRef:
+    from:
+      name: tutorial-vpc
+  routeTableRefs:
+  - from:
+      name: tutorial-public-route-table
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: Subnet
+metadata:
+  name: tutorial-private-subnet1
+spec:
+  availabilityZone: us-west-2a
+  cidrBlock: 10.0.128.0/20
+  vpcRef:
+    from:
+      name: tutorial-vpc
+  routeTableRefs:
+  - from:
+      name: tutorial-private-route-table-az1
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: SecurityGroup
+metadata:
+  name: tutorial-security-group
+spec:
+  description: "ack security group"
+  name: tutorial-sg
+  vpcRef:
+     from:
+       name: tutorial-vpc
+  ingressRules:
+    - ipProtocol: tcp
+      fromPort: 22
+      toPort: 22
+      ipRanges:
+        - cidrIP: "0.0.0.0/0"
+          description: "ingress"
+```
+ 
+The output should look similar to:
+```
+vpc.ec2.services.k8s.aws/tutorial-vpc created
+internetgateway.ec2.services.k8s.aws/tutorial-igw created
+natgateway.ec2.services.k8s.aws/tutorial-natgateway1 created
+elasticipaddress.ec2.services.k8s.aws/tutorial-eip1 created
+routetable.ec2.services.k8s.aws/tutorial-public-route-table created
+routetable.ec2.services.k8s.aws/tutorial-private-route-table-az1 created
+subnet.ec2.services.k8s.aws/tutorial-public-subnet1 created
+subnet.ec2.services.k8s.aws/tutorial-private-subnet1 created
+securitygroup.ec2.services.k8s.aws/tutorial-security-group created
+```
+ 
+* Check the **Custom Resource's** using `kubectl describe`:
```

Comment:
could also add all the crds installed 

`dsaurya@3c063003223c ~ % k get crd
NAME                                         CREATED AT
adoptedresources.services.k8s.aws            2022-10-23T16:35:34Z
dhcpoptions.ec2.services.k8s.aws             2022-10-23T16:35:33Z
elasticipaddresses.ec2.services.k8s.aws      2022-10-23T16:35:33Z
eniconfigs.crd.k8s.amazonaws.com             2022-10-22T00:04:17Z
fieldexports.services.k8s.aws                2022-10-23T16:35:34Z
instances.ec2.services.k8s.aws               2022-10-23T16:35:33Z
internetgateways.ec2.services.k8s.aws        2022-10-23T16:35:33Z
natgateways.ec2.services.k8s.aws             2022-10-23T16:35:33Z
routetables.ec2.services.k8s.aws             2022-10-23T16:35:33Z
securitygrouppolicies.vpcresources.k8s.aws   2022-10-22T00:04:20Z
securitygroups.ec2.services.k8s.aws          2022-10-23T16:35:33Z
subnets.ec2.services.k8s.aws                 2022-10-23T16:35:33Z
transitgateways.ec2.services.k8s.aws         2022-10-23T16:35:33Z
vpcendpoints.ec2.services.k8s.aws            2022-10-23T16:35:34Z
vpcs.ec2.services.k8s.aws                    2022-10-23T16:35:34Z`